compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -166,6 +165,23 @@ function parsePrimaryTypeDesc(Tokenizer tok) returns TypeDesc|err:Syntax {
             // JBUG should not need cast #30191
             return <LeafTypeDesc>cur;
         }
+        "string" => {
```

Comment:
This is not the right way to do it now we have multiple modules. See https://github.com/ballerina-platform/nballerina/issues/493